packages/cli/pipelines/from-s3.ttl

```diff
@@ -0,0 +1,20 @@
+@base         <urn:pipeline:data-cube-curation> .
+@prefix :     <https://pipeline.described.at/> .
+@prefix code: <https://code.described.at/> .
+
+<#LoadCsv>
+  :steps [ :stepList ( _:openCsvFromBucket ) ] ;
+  :variables [ :variable
+    [ a :Variable; :name "S3-ENDPOINT" ] ,
+    [ a :Variable; :name "S3-BUCKET" ]
+  ].
+
+_:openCsvFromBucket
+  a                  :Step ;
```

Comment:
Sorry, I'm not going to nit-pick on this every time, promise. But I don't understand how this formatting works. What are these huge spaces? 

The `LoadCsv` part looks good, but the `openCsvFromBucket` part looks totally different...

Reply:
Haha, sure, I was confused too initially.

So you know how RDF is `S P O` triples. The formatting aligns each of those parts vertically. The large gap is to match the space `code:implementedBy` takes on the next line. Same below

```
code:implementedBy [ code:link <file:../lib/s3#openFile> ;
                     a         code:EcmaScript ] ;
                     ^         ^
```

Reply:
Ok, not sure I like it. As long as it's consistent, I can get used to it.